app/src/main/res/layout/activity_main.xml

```diff
@@ -6,13 +6,49 @@
     android:layout_height="match_parent"
     tools:context=".MainActivity">
 
+    <EditText
+        android:id="@+id/moneyET"
+        android:layout_width="200dp"
```

Comment:
Make the "android:layout_width" attribute to match parent and add margin horizontal instead of making the size fixed to 200dp.